recipes/fgap/meta.yaml

```diff
@@ -0,0 +1,24 @@
+about:
+  home: https://github.com/pirovc/fgap
+  license: MIT
+  summary: "FGAP: an automated gap closing tool"
+build:
+  number: 0
+  skip: True  # [osx]
+  
+package:
+  name: fgap
+  version: 1.8.1
+source:
+  fn: fgap-1.8.1.tar.gz
+  # this is nickp60's fork because bioconda needs tagged releases
```

Comment:
It's totally fine to use Git archives for certain commits, e.g., for this project that would be:
```yaml
  url: https://github.com/pirovc/fgap/archive/e3004998f9058e8505b7812544d4c1dbd595a169.tar.gz
```
That way you won't have to maintain a fork only to have tags.
Additionally, there is also nothing wrong by asking upstream if they could tag their releases in the future :wink:

Reply:
@mbargull ooo, I did not know about Git archives!  I move to that until upstream makes a release.  thanks for the tip!